ConfigSpace/hyperparameters.py

```diff
@@ -579,20 +688,21 @@ def __eq__(self, other):
         else:
             return False
 
-    def to_uniform(self, z=3):
+    # todo check if conversion should be done in initiation call or inside class itsel
```

Comment:
At least it should be np.round instead of int. Otherwise it's probably fine here.